### DISCUSSION TITLE
Ya no se que hacer

### DISCUSSION THREAD

**Discussion by FelipeBENIITEZ** [Launch-X-Latam/MisionFrontEnd#297] [General]
llevo tiempo tratando de ver las grabaciones pero al entrar en los links me pide que inicie sesion y al ingresar los datos de mi cuenta innovaccion me dice que la contraseña expiro

**Comment by CometArao** [comment]
Contáctate por un correo con el equipo de innovacción, responden rápido y te reestablecerán la contraseña. No sé por qué está pasando, pero ya es segunda vez que debo recuperarla jaja, pero te darán una solución.

El correo es: contacto@innovaccion.mx

**Reply by FelipeBENIITEZ** [reply]
ya me la dieron gracias, ojala no me vuelva a cerrar sesion

**Reply by Standard-IO** [reply]
Una pregunta, ¿pierden la contraseña o se les bloquea la cuenta?. Yo las guardo en un gestor de contraseñas para evitar equivocarme al ponerla o olvidarla.
